pkg/infrastructure/powervs/clusterapi/powervs.go

```diff
@@ -163,6 +163,19 @@ func (p Provider) InfraReady(ctx context.Context, in clusterapi.InfraReadyInput)
 	if err != nil {
 		return fmt.Errorf("failed to add ping security group rule: %w", err)
 	}
+
+	if in.InstallConfig.Config.Publish == types.InternalPublishingStrategy &&
+		(len(in.InstallConfig.Config.ImageDigestSources) > 0 || len(in.InstallConfig.Config.DeprecatedImageContentSources) > 0) {
```

Comment:
Assuming we are within the deprecation window of `InstallConfig.Config.DeprecatedImageContentSources` would it be useful to set InstallConfig.Config.ImageDigestSources to InstallConfig.Config.DeprecatedImageContentSources when only InstallConfig.Config.DeprecatedImageContentSources is provided while validating InstallConfig? That way we don't have to check both values every time. 
When you actually stop support for InstallConfig.Config.DeprecatedImageContentSources, you wouldn't have to make changes in so many places.